### PR TITLE
PR for SRVCOM-3431: Add release notes entries in Serverless version 1.34.1

### PR DESCRIPTION
**Pull request by kaldesai**
**Affecting version:** 
`serverless-docs-1.34`
`serverless-docs-1.35`

**Tracking JIRAs:** https://issues.redhat.com/browse/SRVCOM-3431

**Doc previews:** [Red Hat OpenShift Serverless 1.34.1](https://85382--ocpdocs-pr.netlify.app/openshift-serverless/latest/about/serverless-release-notes.html#serverless-rn-1-34-1_serverless-release-notes)

### CODE REVIEW

**Comment by mgencur** [comment]
I'm curious if we also want to include something like this?
```
This release of OpenShift Serverless addresses Common Vulnerabilities and Exposures (CVEs).
```
The changes were only in base images so I'm not sure.

**Comment by mgencur** [comment]
LGTM from QE

File: modules/serverless-rn-1-34-1.adoc [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+= Red Hat {ServerlessProductName} 1.34.1
+
+{ServerlessProductName} 1.34.1 is now available. This release of {ServerlessProductName} addresses identified Common Vulnerabilities and Exposures (CVEs). Fixed issues that pertain to {ServerlessProductName} on {ocp-product-title} are included in the following notes:
+
+[id="fixed-issues-1-34-1_{context}"]
+== Fixed issues
+
+* Previously, requests to Knative Services with slow back-end responses could fail due to the default OpenShift Route timeout being too short. This issue has been addressed by making the `haproxy.router.openshift.io/timeout` setting configurable for automatically created routes.
++
+You can now adjust the timeout by setting the `ROUTE_HAPROXY_TIMEOUT` environment variable in the {ServerlessOperatorName} Subscription configuration as follows:
```
Comment (xenolinux) [inline modules/serverless-rn-1-34-1.adoc:16]:
```suggestion
You can now adjust the timeout by setting the `ROUTE_HAPROXY_TIMEOUT` environment variable in the {ServerlessOperatorName} subscription configuration as follows:
```

OR `Subscription` (with back ticks) configuration

File: modules/serverless-rn-1-34-1.adoc [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  ...
+spec:
+  channel: stable
+  config:
+    env:
+      - name: ROUTE_HAPROXY_TIMEOUT
+        value: '900'
+----
+
+* Previously, long-running requests to Knative Services could be prematurely terminated if the Activator component was scaled down by the `HorizontalPodAutoscaler`. This issue has been resolved. The `terminationGracePeriodSeconds` for the Activator is now automatically aligned with the `max-revision-timeout-seconds` setting configured for Knative revisions.
```
Comment (xenolinux) [inline modules/serverless-rn-1-34-1.adoc:32]:
```suggestion
* Previously, long-running requests to Knative Services could be prematurely terminated if the Activator component was scaled down by using the `HorizontalPodAutoscaler` field. This issue has been resolved. The `terminationGracePeriodSeconds` field for the Activator is now automatically aligned with the `max-revision-timeout-seconds` setting configured for Knative revisions.
```

Field or parameter - whatever you feel suitable.

**Review by xenolinux — COMMENTED** [review]
two suggestions; otherwise LGTM

File: modules/serverless-rn-1-34-1.adoc (hunk 1 above)
Comment (kaldesai) [reply modules/serverless-rn-1-34-1.adoc:16]: Added with the `Subscription` back ticks.

File: modules/serverless-rn-1-34-1.adoc (hunk 2 above)
Comment (kaldesai) [reply modules/serverless-rn-1-34-1.adoc:32]: Done.

**Comment by nainaz** [comment]
I am glad we added CVE comment as well. 
LGTM

**Comment by jab-rh** [comment]
serverless-docs-1.35

**Pull request merged**